front/src/containers/Authentication/login.js

```diff
@@ -43,30 +50,54 @@ const LoginScreen = (props) => {
 
   return (
     <Container>
-      <Logo>Globo Plus</Logo>
-
-      <View>
-        <Text>Email</Text>
-        <InputField
-          value={props.values.email}
-          onChangeText={text => props.setFieldValue('email', text)} />
-
-        <Text>Senha</Text>
-        <InputField
-          value={props.values.password}
-          onChangeText={text => props.setFieldValue('password', text)} />
-
-        <Button
-          onPress={props.handleSubmit}
-          title='Login' />
+      <ContainerLogo>
```

Comment:
Create a container around logo, it's really necessary? In the style that you add, is added only a margin, but you can apply this style in own logo, so, you can remove this.